python/ray/tune/__init__.py

```diff
@@ -2,13 +2,30 @@
 from __future__ import division
 from __future__ import print_function
 
+import logging
+import sys
+
 from ray.tune.error import TuneError
 from ray.tune.tune import run_experiments
 from ray.tune.experiment import Experiment
 from ray.tune.registry import register_env, register_trainable
 from ray.tune.trainable import Trainable
 from ray.tune.suggest import grid_search, function, sample_from
 
+
+def _setup_logger():
```

Comment:
Do we also need stream=sys.stdout for rllib?

Reply:
Hm maybe I should just keep it at stderr and then move `logger.info` into `print` for the pretty result